packages/serverless-contracts/src/contracts/cloudFormation/__tests__/fullContractSchema.type.test.ts

```diff
@@ -2,8 +2,9 @@ import { JSONSchema } from 'json-schema-to-ts';
 
 import { FullContractSchemaType } from '../types';
 
-type Check =
-  FullContractSchemaType<'hello'> extends JSONSchema ? 'pass' : 'fail';
+type Check = FullContractSchemaType<'hello'> extends JSONSchema
+  ? 'pass'
+  : 'fail';
```

Comment:
I'm a little bit confused by this change, it should disappear with a `pnpm format-fix` at the repo root

Reply:
I removed this commit after fixing the pnpm.lock (cf https://github.com/swarmion/swarmion/pull/996#issuecomment-2041456965)